.github/ISSUE_TEMPLATE/bug_report_template.md

```diff
@@ -12,9 +12,14 @@ assignees: ''
 Thank you for submitting an issue. Please refer to our [issue policy](https://www.github.com/voxel51/fiftyone/blob/develop/ISSUE_POLICY.md) for information on what types of issues we address.
 
 1. Please fill in this template to ensure a timely and thorough response
-2. Place an "x" between the brackets next to an option if it applies. For example:
+2. Remove the section instructions but leave the section header
```

Comment:
Try to make sure the report is cleaner ... we want reporter to remove the descriptions